libmamba/tests/src/validation/test_update_framework_v0_6.cpp

```diff
@@ -249,7 +251,7 @@ TEST_SUITE("validation::v0_6::RootImpl")
         out_file.close();
 
         // "2.sv1.root.json" is not compatible spec version (spec version N)
-        CHECK_THROWS_AS(v0_6::RootImpl root(p), role_file_error);
+        CHECK_THROWS_AS(v0_6::RootImpl{ p }, role_file_error);
```

Comment:
When defining `DOCTEST_CONFIG_VOID_CAST_EXPRESSIONS` we can't write arbitrary expressions here, we only can write something cast-able to void. I think it's still clear what's going on here, so it's not a big deal.